recipes/mysqlclient/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "mysqlclient" %}
+{% set version = "1.3.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ff8ee1be84215e6c30a746b728c41eb0701a46ca76e343af445b35ce6250644f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - pip
+    - mysql-connector-c
+
+  run:
+    - python
+    - mysql-connector-c
```

Comment:
Doesn't mysql-connector-c have run_exports?

Reply:
Doesn't look like it (see [recipe](https://github.com/conda-forge/mysql-connector-c-feedstock/blob/master/recipe/meta.yaml)).